cal/groups.yml

```diff
@@ -0,0 +1,248 @@
+!!python/unicode '2015 Columbia College Student Council':
```

Comment:
Yaml.dump does this, it's python-specific. There's probably a way to not use these, but I don't think it should be an issue.